data/app.js

```diff
@@ -0,0 +1,15 @@
+const Dirwatcher = require('./dirwatcher.js');
+const Importer = require('./importer.js');
+
+const dirwatcher = new Dirwatcher();
+const importer = new Importer();
+
+const dirPath = '/Users/Yahor_Pamakha/js-workspace/data/';
+
+dirwatcher.on('changed', (filename) => {
+    console.log(`${filename} changed`);
+    importer.import(dirPath + filename);
+    //importer.importSync(dirPath + filename);
+});
+
+dirwatcher.watch(dirPath, 100);
```

Comment:
Consider exercising "separation of concerns" here, making both "dirwatcher" and "importer" and their respective listeners unaware of each other.
In other words, use the same EventEmmiter from above comment that emits an change event as a callback to "watch()" method.